.circleci/config.yml

```diff
@@ -14,6 +14,31 @@ nvm: &nvm
             nvm install $(cat calypso/.nvmrc)
 
 version: 2
+
+references:
+  npm_restore_cache: &npm_restore_cache
+    restore_cache:
+      name: Restore npm cache
+      keys:
+        - v1-npm-{{ arch }}-{{ checksum "calypso/.nvmrc" }}-{{ checksum "package-lock.json" }}-{{ checksum "calypso/package-lock.json" }}
+        - v1-npm-{{ arch }}-{{ checksum "calypso/.nvmrc" }}-{{ checksum "package-lock.json" }}
+        - v1-npm-{{ arch }}-{{ checksum "calypso/.nvmrc" }}
+  npm_install: &npm_install
+    run:
+      name: Npm install
+      command: |
+        source $HOME/.nvm/nvm.sh
+        nvm use
+        npm ci
+        cd calypso
+        npm ci
```

Comment:
This `cd calypso; npm ci` is a bit different in that it's installing the calypso deps each time. Maybe the install steps should be unique.

Reply:
changed to match previous behavior in dd7c00a